### PR TITLE
[gcsweb] add the content-type in headers when handling an object

### PR DESCRIPTION
**Pull request by droslean**
/cc @stevekuznetsov 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: gcsweb/cmd/gcsweb/gcsweb.go [hunk 1]
```diff
@@ -371,7 +373,7 @@ func (s *server) gcsRequest(w http.ResponseWriter, r *http.Request) {
 
 	// This means that the object is a file.
 	if objAttrs != nil {
-		if err := s.handleObject(w, bucket, object); err != nil {
+		if err := s.handleObject(w, bucket, object, objAttrs.ContentType); err != nil {
```
Comment (wking) [inline gcsweb/cmd/gcsweb/gcsweb.go:376]: Probably want to pass through `ContentEncoding` too?

Comment (droslean) [reply gcsweb/cmd/gcsweb/gcsweb.go]: I include also the encoding.

File: gcsweb/cmd/gcsweb/gcsweb_test.go [hunk 2]
```diff
@@ -209,7 +209,7 @@ func TestHandleObject(t *testing.T) {
 
 			s := server{storageClient: client}
 
-			err = s.handleObject(w, tc.bucket, tc.object)
+			err = s.handleObject(w, tc.bucket, tc.object, objectHeaders{})
```
Comment (stevekuznetsov) [inline gcsweb/cmd/gcsweb/gcsweb_test.go:212]: Ideally we would check that headers are set  correctly, too, in the test

Comment (droslean) [reply gcsweb/cmd/gcsweb/gcsweb_test.go:212]: Yea that will need to actually have different file content and check the desired headers as well.

Comment (droslean) [reply gcsweb/cmd/gcsweb/gcsweb_test.go:212]: We will need to change the mock server also to generate the headers as well. I will need to study their docs etc. This sounds like a nice follow-up

Comment (droslean) [reply gcsweb/cmd/gcsweb/gcsweb_test.go]: addressed

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**